docker/third-party/images.txt

```diff
@@ -5,6 +5,12 @@ amundsendev/amundsen-search:2.4.0
 debian:9.5
 ghost:3.0-alpine
 google/cloud-sdk:305.0.0-slim
+hailgenetics/python-dill:3.6
+hailgenetics/python-dill:3.6-slim
+hailgenetics/python-dill:3.7
+hailgenetics/python-dill:3.7-slim
+hailgenetics/python-dill:3.8
+hailgenetics/python-dill:3.8-slim
```

Comment:
this seems a bit odd. These images are defined in docker/python-dill. Maybe we should just add a `gcr.io/..` alias there?

Reply:
Can you expand on this idea more?